services/explorer-api/src/modules/node/node.service.ts

```diff
@@ -30,6 +31,16 @@ export class NodeService {
     };
   }
 
+  public async findByAddress(address: string): Promise<TaraxaNode> {
+    const node = await this.repository.findOneBy({ address: address });

```

Comment:
`findOneOrFail`  instead of if ?

Reply:
Works but I prefer to handle and throw my own expections in cases like this. That`s why I used this

Reply:
OK. W/e you prefer